docs/release/trg-0/trg-7-1.md

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Overall Fabian already changed all md trg files by adding a zero: trg-7-01.md due to me adding this format in TRG 5.* because as soon as you go over 10, the order is broken:

5.1
5.10
5.2